pkg/pillar/devicenetwork/dnc.go

```diff
@@ -232,20 +232,20 @@ func VerifyPending(ctx *DeviceNetworkContext, pending *DPCPending,
 		checkAndUpdateWireless(ctx, &pending.OldDPC, &pending.PendDPC)
 
 		log.Infof("VerifyPending: DPC changed. update DhcpClient.\n")
-		ifname, err := UpdateDhcpClient(pending.PendDPC, pending.OldDPC)
-		if err != nil {
+		intfStatusMap := UpdateDhcpClient(pending.PendDPC, pending.OldDPC)
+		pending.PendDPC.UpdatePortStatusFromIntfStatusMap(intfStatusMap)
+		if !intfStatusMap.HasSuccessfulIntf() {
```

Comment:
Can you explain how this check is different than the old check?
From what I can tell it is never successful, since you don't record successes any more.

Reply:
Yeah.. The change is not fully done. As I said, the algo needs to change to:

1) if interface has error -wait
2) If no errors - or end of wait time - check if atleast 1 interface without errors - and proceed